packages/react-dropdown/src/MultiDownshift.js

```diff
@@ -41,14 +41,10 @@ type State = {
 };
 
 class MultiDownshift extends React.Component<Props, State> {
-  constructor(props: Props) {
```

Comment:
constructors are hard to write and read, since 99% of the cases can be covered by class properties I'd tend to avoid to use them